sway-core/src/asm_generation/fuel/analyses.rs

```diff
@@ -17,9 +17,9 @@ use crate::asm_lang::{ControlFlowOp, Label, Op, VirtualRegister};
 /// Two tables are generated: `live_in` and `live_out`. Each row in the tables corresponds to an
 /// instruction in the program.
 /// * A virtual register is in the `live_out` table for a given instruction if it is live on any
-/// of that node's out-edges
+///   of that node's out-edges
 /// * A virtual register is in the `live_in` table for a given instruction if it is live on any
-/// of that node's in-edges
+///   of that node's in-edges
```

Comment:
looks like there's a ton of these new spaces throughout the comments, just wondering how they got there?

Reply:
ok out of curiosity which lint is this? Would love to understand what the reason behind it is. Seems a bit off at first glance.

Reply:
Ah, sorry my mistake... I just reverse those changes locally and it still passes tests. I must have changed these all to silence Clippy:

```
warning: doc list item without indentation
  --> sway-core/src/asm_generation/fuel/analyses.rs:20:5
   |
20 | /// of that node's out-edges
   |     ^
   |
   = help: if this is supposed to be its own paragraph, add a blank line
   = help: for further information visit https://rust-lang.github.io/rust-clippy/master/index.html#doc_lazy_continuation
   = note: `#[warn(clippy::doc_lazy_continuation)]` on by default
help: indent this line
   |
20 | ///   of that node's out-edges
   |     ++
```

Before this change, clippy was full of this same message which was a distraction.

I guess we could `allow(clippy::doc_lazy_continuation)` instead, but this change looks cleaner to be in the editor because you can see at a glance that it's not just more paragraph text.

Reply:
Ahh!! It turns out it was an error after all - `clippy::doc_lazy_continuation` was introduced in `1.80.0` and when I tried bumping Rust to `1.81.0`, the `cargo-clippy` GitHub Action workflow started failing - for some reason they made it an `error` rather than a `warning` 🫠

Reply:
Nice! So apart from that Clippy error, upgrading Rust to `1.81.0` has no other regressions.